guides/source/configuring.md

```diff
@@ -455,11 +455,11 @@ to `'http authentication'`.
 * `config.action_dispatch.signed_cookie_salt` sets the signed cookies salt value.
 Defaults to `'signed cookie'`.
 
-* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt
-value. Defaults to `'encrypted cookie'`.
+* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
 
-* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed
-encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
+* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
+
+* `config.action_dispatch.authenticated_encrypted_cookie_salt` sets the authenticated encrypted cookie salt. Defaults to `'authenticated encrypted cookie'`.
```

Comment:
I'm really confused about this. Are apps expected to overwrite this? When would it make sense for apps to do so?